server/server.go

```diff
@@ -388,6 +388,12 @@ func NewServer(opts *Options) (*Server, error) {
 	// Ensure that non-exported options (used in tests) are properly set.
 	s.setLeafNodeNonExportedOptions()
 
+	// Setup OCSP Stapling. This will abort server from starting if there
+	// are no valid staples and OCSP policy is to Always or MustStaple.
+	if err := s.enableOCSP(); err != nil {
```

Comment:
enableOCSP had to be done when NewServer is called instead of Start, otherwise gateway remotes TLS callbacks were not properly configured